WORKSPACE

```diff
@@ -297,3 +297,11 @@ http_archive(
     strip_prefix = "clang+llvm-8.0.0-x86_64-linux-gnu-ubuntu-18.04",
     url = "http://releases.llvm.org/8.0.0/clang+llvm-8.0.0-x86_64-linux-gnu-ubuntu-18.04.tar.xz",
 )
+
+# Roughtime
+http_archive(
+    name = "roughtime",
+    build_file = "@//third_party/roughtime:roughtime.BUILD",
+    #sha256 = "47c99712b4b34a6713ea7ecfec6860e6b88ba046df206386346de4c3409fe6dd",
```

Comment:
un-comment this?

Reply:
This is a bit of a problem. Each time this is downloaded it generates a new archive, so the hash changes. I will look into other options, such as using new_git_repository.

Reply:
Changed to using new_git_repository with shallow_since to make it deterministic.